bin/test-example

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# need bash, not sh, for default variable ${var:-default}
+
+exampledir=$1
+
+examplename=$(basename "$exampledir")
+exercisedir=$(dirname $(dirname "$exampledir"))
```

Comment:
I would like t use `realpath`, but it doesn't exist on travis...???

The effect of not using `realpath` is that you cannot say in an `examples` dir `test-example success-standard` because then `exercisedir` becomes `.` instead of `..` as it should be.